app/javascript/components/common/Accordion.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+export type AccordionContext = {
+  index: string
+  open: boolean
```

Comment:
One small suggestion: maybe `isOpen` would be a more descriptive name? `open` sounds like it could also be a method. Just a suggestion though.

Reply:
Cool. I've used `isOpen`.